index.html

```diff
@@ -9,31 +9,45 @@
     <script src="https://kit.fontawesome.com/091ffb7972.js" crossorigin="anonymous"></script>
 </head>
 <body>
+  <div class="background-desktop">
   <header class="head">
-    <a href="#" class="pagename" >Welcome</a>
-    <nav>
+    <nav class="nav-bar">
+      <div class="mobile-navhead">
+      <a href="#" class="pagename" >Welcome</a>
       <img src="tools/Union.svg" alt="" class="nav-icon">
+    </div>
+      <ul class="desktop-nav-container">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
     </nav>
   </header>
   <!-- welcome page -->
     <section class="welcome">
         <h1><span class="greetings">Hey there. I’m Josue</span><br>
         <span class="profession">I’m a software developer</span></h1>
-        <p class="description">I'm a student who is in his first week of the microverse module,
-             this is my first project I'm working on to see my advencements or progress.</p>
+        <p class="description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
```

Comment:
- [ ] While making changes requested by the previous reviewer, you probably forgot to put this into consideration. You will notice that the content paragraph has shifted down and the gap between the three text tags is too wide. It was not like that before the previous reviewer required changes. Kindly bring it up a little so that it will fit the Figma. Kindly reference the Figma design posted by the previous reviewer.

### Your work before the previous reviewer required changes
![image](https://user-images.githubusercontent.com/86200356/200488771-dec22847-6ee9-4a26-b9a8-78a7af03a96c.png)

### Your work now
![image](https://user-images.githubusercontent.com/86200356/200489033-a0c4b740-79e4-4717-836b-f4bcc62f78f2.png)

Reply:
@Olamarx can you be more specific

Reply:
@ijosue16 I am talking about the 'I can help you build a product...', kindly make the text go up so that it would look like what it was before the previous reviewer required changes.

Do you understand, please?

Reply:
@Olamarx thank you for clearing it out.

Reply:
You are welcome 😊